core/server/utils/startup-check.js

```diff
@@ -227,6 +229,49 @@ checks = {
             console.error('\x1B[31mWARNING: Ghost is attempting to use a direct method to send email. \nIt is recommended that you explicitly configure an email service.\033[0m');
             console.error('\x1B[32mHelp and documentation can be found at http://support.ghost.org/mail.\033[0m\n');
         }
+    },
+
+    builtFilesExist: function builtFilesExist() {
+        var configFile,
+            config,
+            location,
+            fileNames = ['ghost.js', 'vendor.js', 'ghost.css', 'vendor.css'];
+
+        try {
+            configFile = require(configFilePath);
+            config = configFile[mode];
+
+            if (config.paths && config.paths.clientAssets) {
+                location = config.paths.clientAssets;
+            } else {
+                location = path.join(appRoot, '/core/built/assets/');
+            }
+        } catch (e) {
+            location = path.join(appRoot, '/core/built/assets/');
+        }
+
+        if (process.env.NODE_ENV === 'production') {
+            // Production uses `.min` files
+            fileNames = fileNames.map(function (file) {
+                return file.replace('.', '.min.');
+            });
+        }
+
+        function checkExist(fileName) {
+            try {
+                fs.statSync(fileName);
+            } catch (e) {
+                console.error('\x1B[31mERROR: Unable to open some of the built files!\033[0m');
```

Comment:
Can we have this error message be the same as it was before? https://github.com/TryGhost/Ghost/blob/4ba0a745dfb8591e36f45002ba517e9da62bc7b8/core/server/translations/en.json#L189-L189

Also, these messages aren't used anywhere else so they can be removed from en.json